packetbeat/protos/tls/_meta/fields.yml

```diff
@@ -621,14 +621,6 @@
       type: alias
       path: tls.detailed.server_certificate.issuer.locality
 
-    - name: tls.server_certificate_chain
-      type: alias
-      path: tls.detailed.server_certificate_chain
-
-    - name: tls.client_certificate_chain
-      type: alias
-      path: tls.detailed.client_certificate_chain
-
```

Comment:
These aliases need to be removed because it's not possible to point an alias to an object.